MonoGame.Framework.Content.Pipeline/IContentContext.cs

```diff
@@ -0,0 +1,14 @@
+namespace Microsoft.Xna.Framework.Content.Pipeline;
```

Comment:
If this in a non XNA class it should probably be in a MonoGame namespace, especially as it adds to the public api.